packages/styles/src/calendar.scss

```diff
@@ -126,178 +40,129 @@ $fd-calendar-special-days: (
 );
 
 @mixin special-days($nameColorMap) {
-  @each $name, $color in $nameColorMap {
-    &__special-day--#{$name} {
+  &__special-day {
+    .#{$block}__text::after {
```

Comment:
`.#{$block}__text::after` is repeated many times. Maybe put it in a variable where `.#{$block}` is defined.

```
$block: #{$fd-namespace}-calendar;
$specialDay: #{$fd-namespace}-calendar__text::after;
```

then you can use it like this: `.#{$specialDay}`